s3-streamer

```diff
@@ -164,8 +166,9 @@ class ChunkedUploader:
         self.pending = b''
         self.send_at = 0  # Send the first write immediately
 
-    def init(self):
-        self.destination.write(f'{self.filename}.chunks', json.dumps([]).encode('ascii'))
+    def start(self, data):
+        # Send the initial data immediately, to get the chunks file written out.
+        self.append_block(data.encode('utf=-8'))
```

Comment:
That looks like a typo. Does that actually work?

Reply:
That looks like a typo.

Reply:
For the record, I did test it.